Documentation/ApiOverview/Seo/XmlSitemap.rst

```diff
@@ -1,263 +1,299 @@
-.. include:: /Includes.rst.txt
-.. index:: XML sitemap
-.. _xmlsitemap:
+..  include:: /Includes.rst.txt
+..  index:: XML sitemap
+..  _xmlsitemap:
 
 ===========
 XML sitemap
 ===========
 
-.. versionadded:: 9.4
-
-   XML Sitemap support was added to the Core with change :doc:`ext_core:Changelog/9.4/Feature-84525-XMLSitemap`.
-
-It is possible to generate XML sitemaps for SEO purposes without using 3rd-party plugins.
-When enabled, this new feature will create a sitemapindex with one or more sitemaps in it.
-Out-of-the-box it will have one sitemap containing all the pages of the current site and
-language. Per site and per language you have the possibility to render a different sitemap.
+It is possible to generate XML sitemaps for SEO purposes without using 3rd party
+plugins. When this feature is enabled, a sitemap index file is created with one
+or more sitemaps in it. By default, there will be one sitemap that contains all
+pages of the current site and language. Per each site and language, you can
+render a different sitemap.
 
 .. note::
    The XML sitemap is provided by the optional system extension
    EXT:seo. You can find information about how to install and use it in the
    :doc:`EXT:seo manual <ext_seo:Index>`.
 
+.. contents:: Table of Contents
+   :depth: 1
+   :local:
+
 Installation
 ============
 
-The XML sitemap is disabled by default. You can easily enable it by installing the system
-extension "seo" and including the static TypoScript template XML Sitemap (seo). It is also
-mandatory to have a site configuration for your rootpage(s).
+The XML sitemap is not available by default. You can easily activate it by
```

Comment:
What do you consider default? In the TYPO3/DEFAULT package of the composer helper seo is already contained and activated: https://get.typo3.org/misc/composer/helper

Reply:
I considered typo3/minimal: https://packagist.org/packages/typo3/minimal

Reply:
Maybe: "The seo system extension provides XML sitemaps. If it is not available on your installation, you can easily ..."

Reply:
Better, but avoid easily, this frustrates people who will have trouble installing